contain-intrinsic-* missing properties (#21110)
mdn/content: files/en-us/web/css/contain-intrinsic-width/index.md

---
title: contain-intrinsic-width
slug: Web/CSS/contain-intrinsic-width
tags:
  - CSS
  - CSS Property
  - Reference
  - experimental
  - contain-intrinsic-width
browser-compat: css.properties.contain-intrinsic-width
---

{{CSSRef}} {{SeeCompatTable}}

> The **`contain-intrinsic-width`** [CSS](/en-US/docs/Web/CSS) property sets the width of an element that will be used for layout when it is subject to [size containment](/en-US/docs/Web/CSS/CSS_Containment#size_containment).

Reviewer: ```suggestion
The **`contain-intrinsic-width`** [CSS](/en-US/docs/Web/CSS) property sets the width of an element that a browser will use for layout when the element is subject to [size containment](/en-US/docs/Web/CSS/CSS_Containment#size_containment).
```
